site/examples/map_components.js

```diff
@@ -0,0 +1,86 @@
+  <strong class='block mb6'>Title description</strong>
+  <div class='flex-parent mb6'>
+    <div class='flex-child flex-child--grow bg-blue-faint h12'></div>
+    <div class='flex-child flex-child--grow bg-blue-light h12'></div>
+    <div class='flex-child flex-child--grow bg-blue h12'></div>
+    <div class='flex-child flex-child--grow bg-blue-dark h12'></div>
+    <div class='flex-child flex-child--grow bg-red-faint h12'></div>
+    <div class='flex-child flex-child--grow bg-red-light h12'></div>
+    <div class='flex-child flex-child--grow bg-red h12'></div>
+    <div class='flex-child flex-child--grow bg-red-dark h12'></div>
+  </div>
+  <div class='flex-parent txt-xs'>
+    <div class='flex-child flex-child--grow'>Low</div>
+    <div class='flex-child flex-child--grow align-r'>High</div>
+  </div>
+</div>`;
+
+const basicLegendTwo = `<div class='w240 round shadow-darken5 p12 txt-s'>
+  <strong class='block mb6'>Title description</strong>
+  <div class='flex-parent mb6'>
+    <div class='flex-child flex-child--grow bg-green-faint h12'></div>
+    <div class='flex-child flex-child--grow bg-green-light h12'></div>
+    <div class='flex-child flex-child--grow bg-green h12'></div>
+    <div class='flex-child flex-child--grow bg-green-dark h12'></div>
+  </div>
+  <div class='flex-parent txt-xs'>
+    <div class='flex-child flex-child--grow align-center'>0</div>
+    <div class='flex-child flex-child--grow align-center'>25</div>
+    <div class='flex-child flex-child--grow align-center'>50</div>
+    <div class='flex-child flex-child--grow align-center'>75</div>
+  </div>
+</div>`;
+
+const radiusLegend = `<div class='w240 round shadow-darken5 p12 txt-s'>
+  <strong class='block mb6'>Title description</strong>
+  <div class='flex-parent flex-parent--center-main flex-parent--center-cross mb6'>
+    <div class='flex-child flex-child--grow align-center'>
+      <span class='inline-block w3 h3 round-full bg-blue'></span>
+    </div>
+    <div class='flex-child flex-child--grow align-center'>
+      <span class='inline-block w6 h6 round-full bg-blue'></span>
+    </div>
+    <div class='flex-child flex-child--grow align-center'>
+      <span class='inline-block w12 h12 round-full bg-blue'></span>
+    </div>
+    <div class='flex-child flex-child--grow align-center'>
+      <span class='inline-block w18 h18 round-full bg-blue'></span>
+    </div>
+    <div class='flex-child flex-child--grow align-center'>
+      <span class='inline-block w24 h24 round-full bg-blue'></span>
+    </div>
+  </div>
+  <div class='flex-parent txt-xs'>
+    <div class='flex-child flex-child--grow align-center'>3px</div>
+    <div class='flex-child flex-child--grow align-center'>6px</div>
+    <div class='flex-child flex-child--grow align-center'>12px</div>
+    <div class='flex-child flex-child--grow align-center'>18px</div>
+    <div class='flex-child flex-child--grow align-center'>24px</div>
+  </div>
+</div>`;
```

Comment:
The horizontal alignment looks off to me:

![screen shot 2017-01-17 at 11 45 42 am](https://cloud.githubusercontent.com/assets/628431/22034768/88a750a6-dcaa-11e6-89f6-30b91aea31a0.png)